.github/workflows/ci-docs.yaml

```
@@ -0,0 +1,34 @@
+name: Documentation
+permissions: read-all
+
+on:
+  pull_request:
+    branches:
+      - main
+      - git_push_stack/**
+
+jobs:
+  docs:
+    timeout-minutes: 5
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.9.5
```

```suggestion
          python-version: 3.9
```
I don't think we need to me that precise